lms/djangoapps/course_home_api/course_metadata/views.py

```diff
@@ -139,6 +140,7 @@ def get(self, request, *args, **kwargs):
             'can_view_certificate': certificates_viewable_for_course(course),
             'course_modes': course_modes,
             'is_new_discussion_sidebar_view_enabled': new_discussion_sidebar_view_is_enabled(course_key),
+            'has_course_author_access': has_course_author_access(request.user, course_key, 'cms'),
```

Comment:
I think we should leave a comment here explaining why we had to pass `cms` as the service variant here, even though the API might have been called from the LMS